From 4602f117c9017f29383bd38efd28332c5c2d3996 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Mon, 7 Jan 2019 20:38:09 +0200
Subject: [PATCH 1/1] change eac3 mime type

- aiurdemux eac3 mime type should be audio/x-eac3
- beepdec sink caps update to audio/x-eac3

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 plugins/aiurdemux/aiurdemux.c                 | 2 +-
 plugins/beepdec/beep_registry_1.0.arm.cf.in   | 2 +-
 plugins/beepdec/beep_registry_1.0.arm12.cf.in | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/plugins/aiurdemux/aiurdemux.c b/plugins/aiurdemux/aiurdemux.c
index 38f53cf..7dbe7e6 100755
--- a/plugins/aiurdemux/aiurdemux.c
+++ b/plugins/aiurdemux/aiurdemux.c
@@ -2458,7 +2458,7 @@ static void aiurdemux_parse_audio (GstAiurDemux * demux, AiurDemuxStream * strea
         codec = "Dobly Digital Plus (E-AC3)";
         mime =
             g_strdup_printf
-            ("audio/eac3, channels=(int)%ld, rate=(int)%ld, bitrate=(int)%ld",
+            ("audio/x-eac3, channels=(int)%ld, rate=(int)%ld, bitrate=(int)%ld",
             stream->info.audio.n_channels, stream->info.audio.rate,
             stream->bitrate);
         break;
diff --git a/plugins/beepdec/beep_registry_1.0.arm.cf.in b/plugins/beepdec/beep_registry_1.0.arm.cf.in
index 86bde83..c3a9bd3 100755
--- a/plugins/beepdec/beep_registry_1.0.arm.cf.in
+++ b/plugins/beepdec/beep_registry_1.0.arm.cf.in
@@ -37,7 +37,7 @@ library = @vorbisd_arm_SONAME@
 [ddplus]
 longname = DDplus decoder
 description = Decode Dobly Digital Plus audio to raw data
-mime = audio/eac3
+mime = audio/x-eac3
 library = @ddpd_arm_SONAME@
 
 [amr-nb]
diff --git a/plugins/beepdec/beep_registry_1.0.arm12.cf.in b/plugins/beepdec/beep_registry_1.0.arm12.cf.in
index e4e2172..bd64132 100755
--- a/plugins/beepdec/beep_registry_1.0.arm12.cf.in
+++ b/plugins/beepdec/beep_registry_1.0.arm12.cf.in
@@ -35,7 +35,7 @@ library = @vorbisd_arm12_SONAME@
 [ddplus]
 longname = DDplus decoder
 description = Decode Dobly Digital Plus audio to raw data
-mime = audio/eac3
+mime = audio/x-eac3
 library = @ddpd_arm12_SONAME@
 
 [amr-nb]
-- 
2.7.4

